examples/README.md

```diff
@@ -0,0 +1,20 @@
+# Python Client Examples
+
+This directory contains various examples how to use the Python client. Please
+read the description at the top of each script for more information about what
+it does and any prequisite steps. Most scripts also include comments throughout
+the code.
+
+## Setup
+
+These scripts require Python 3.5+ and the Kubernetes client which can be
```

Comment:
Python 2.7 is also OK

Reply:
Fixed this, will push on the next commit after more review comments.